Tools/_sunrise/changelog/actions_changelogs_since_last_run.py

```
@@ -161,18 +169,24 @@ def send_embed_discord(embed: dict) -> None:
+            return
+        if response.status_code == 429 and retry_count < DISCORD_RETRY_LIMIT:
             retry_after = response.json().get("retry_after", 1)
             print(f"Rate limited: sleeep {retry_after} seconds")
             time.sleep(retry_after)
-        else:
-            print(f"Failed to send message to Discord: {response.status_code} {response.text}")
-            break
+            continue
+
+        response.raise_for_status()
+        raise RuntimeError(f"Discord webhook вернул неожиданный статус {response.status_code}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)